app/views/results/index.html.erb

```diff
@@ -43,6 +43,41 @@
           </p>
         </div>
       </div>
+      <% unless params[:l] == "3" %>
```

Comment:
Can we make this a method in `ResultsView`? Then we can test it more easily and name it so we know what `params[:l] == "3"` means.